integration_tests/e2e/prisons/configuration/prisonContactDetails.cy.ts

```diff
@@ -0,0 +1,77 @@
+import TestData from '../../../../server/routes/testutils/testData'
+import Page from '../../../pages/page'
+import PrisonConfigPage from '../../../pages/prisons/configuration/prisonConfig'
+import PrisonContactDetailsPage from '../../../pages/prisons/configuration/prisonContactDetails'
+
+context('Prison configuration - contact details', () => {
```

Comment:
Add new tests for prison contact details - think I forgot to do these when implementing the feature initially...